lib/manager/flux/index.ts

```diff
@@ -1,8 +1,36 @@
 import { HelmDatasource } from '../../datasource/helm';
+import { logger } from '../../logger';
+import { exec } from '../../util/exec';
+import type { ExecOptions } from '../../util/exec/types';
+//import { doAutoReplace } from '../../workers/branch/auto-replace';
+import type { UpdateDependencyConfig } from '../types';
 export { extractAllPackageFiles, extractPackageFile } from './extract';
 
 export const defaultConfig = {
   fileMatch: [],
 };
 
 export const supportedDatasources = [HelmDatasource.id];
+
+export async function updateDependency({
```

Comment:
Alright, I initially started with that approach since I saw it in the `gradle-wrapper` manager, but then I thought maybe `updateDependency` was more relevant since it seemed to be responsible for updating a single file's contents (which is all we need to do for Flux - a single file contains all of the system resources that need to be updated). But if `updateArtifacts` would be better, I'll switch back to that approach.

Reply:
Please use `updateArtifacts`

Reply:
and add it to `./artifacts.ts` 😉

Reply:
Alright, I switched back to the `updateArtifacts` approach - if the general idea is sound now, I'll clean it up, add tests and docs, etc.

Reply:
Which approach would you prefer for testing `artifacts.ts`? Should I mock `exec`? Or assume `flux` is installed and run a legit e2e test? I see both approaches in other managers.

Reply:
Looks like the CI tests don't run in a container with `flux` installed so I'll go the mock route.

Reply:
Yes, mock exec calls, checkout other managers for samples